Jenkinsfile

```diff
@@ -16,22 +16,29 @@ node('openshift-build-1') {
     timeout(time: 30, unit: 'MINUTES') {
       deleteDir()
       set_workspace()
-      stage('clone') {
-        dir('aos-cd-jobs') {
+      dir('aos-cd-jobs') {
+        stage('clone') {
           checkout scm
           sh 'git checkout master'
         }
-      }
-      stage('run') {
-        sshagent(['openshift-bot']) { // git repo privileges stored in Jenkins credential store
-          sh '''\
-virtualenv env/
-. env/bin/activate
+        stage('run') {
+          final url = sh(
+            returnStdout: true,
+            script: 'git config remote.origin.url')
+          def prune = 'true', key = 'openshift-bot'
+          if(url.trim() != 'git@github.com:openshift/aos-cd-jobs.git') {
+            prune = ''
```

Comment:
'false'? Seems odd to mix 'true' and empty string.

Reply:
That's an old, shell style boolean.  But I agree it's not necessary here.
Changed to something better.